ibc_data/contrasts/VisualSearch.csv

```diff
@@ -0,0 +1,14 @@
+condition,probe_item–trial_onset,search_array–trial_onset,probe_item_absent–probe_item_present,search_array_absent–search_array_present ,probe_item_four–probe_item_two,search_array_four–search_array_two,delay_vis–delay_wm

```

Comment:
The `probe_item_four–probe_item_two` contrast is defined like this in the doc:

Probe item events (both present and absent) when memory array of size four is presented first vs. probe item events (both present and absent) when memory array of size two is presented first.

I am unsure on how to weight the array that is presented first.

Reply:
They have not been defined in https://github.com/hbp-brain-charting/public_analysis_code/blob/master/ibc_public/utils_contrasts.py#L550
 Could you open a separate issue about that ?